xbmc/interfaces/AnnouncementManager.cpp

```diff
@@ -134,9 +136,9 @@ void CAnnouncementManager::DoAnnounce(AnnouncementFlag flag, const char *sender,
     announcers[i]->Announce(flag, sender, message, data);
 }
 
-void CAnnouncementManager::DoAnnounce(AnnouncementFlag flag, const char *sender, const char *message, CFileItemPtr item, CVariant &data)
+void CAnnouncementManager::DoAnnounce(AnnouncementFlag flag, const char *sender, const char *message, const CFileItemPtr item, const CVariant &data)
 {
-  if (!item.get())
+  if (item == nullptr)
```

Comment:
@paxxi Just curious how this works internally? what does the compiler make out of this?